### PR TITLE
add error

### PR DESCRIPTION
**Pull request by keangkai**
add error function if wrong Am sorry

## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>

<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

### CODE REVIEW

**Comment by brianamarie** [comment]
@keangkai Thank you for opening a pull request! 

@stoe @JasonEtco Do you have the bandwidth to review this in the next week?

**Comment by keangkai** [comment]
Thank you.If wrong sorry.And edit it better

File: _javascript/curriculum.js [hunk 1]
```diff
@@ -94,6 +94,9 @@ $(function(){
 
         updateSlideSize();
       }
+     //add error from ajax call
+      error: function (xhr) {
```
Comment (stoe) [inline _javascript/curriculum.js]: What is the error method used for if it does not have a body?
Not having it fulfills the same purpose.

Comment (brianamarie) [reply _javascript/curriculum.js]: @JasonEtco Do you have any ideas here?

Comment (JasonEtco) [reply _javascript/curriculum.js]: What @stoe said -  not having the method at all is functionally the same as having an empty one.

**Comment by brianamarie** [comment]
@keangkai I'm going to close this pull request, but this is a good idea! It would be good to have an error handler but since the function is empty, the error handler doesn't do anything. If you'd like to work more, please do open another pull request. If you have any questions please let us know. 😄

**Pull request closed without merging**